src/vm/contexts.rs

```diff
@@ -201,6 +290,74 @@ impl AssetMap {
 
         return map
     }
+
+    pub fn get_stx(&self, principal: &PrincipalData) -> Option<u128> {
+        match self.stx_map.get(principal) {
+            Some(value) => Some(*value),
+            None => None
+        }
+    }
+
+    pub fn get_stx_burned(&self, principal: &PrincipalData) -> Option<u128> {
+        match self.burn_map.get(principal) {
+            Some(value) => Some(*value),
+            None => None
+        }
+    }
+
+    pub fn get_stx_burned_total(&self) -> u128 {
+        let mut total : u128 = 0;
+        for principal in self.burn_map.keys() {
+            total = total.checked_add(*self.burn_map.get(principal).unwrap_or(&0u128)).expect("BURN OVERFLOW");
+        }
+        total
+    }
+
+    pub fn get_fungible_tokens(&self, principal: &PrincipalData, asset_identifier: &AssetIdentifier) -> Option<u128> {
+        match self.token_map.get(principal) {
+            Some(ref assets) => match assets.get(asset_identifier) {
+                Some(value) => Some(*value),
+                None => None,
+            },
+            None => None
+        }
+    }
+    
+    pub fn get_nonfungible_tokens(&self, principal: &PrincipalData, asset_identifier: &AssetIdentifier) -> Option<&Vec<Value>> {
+        match self.asset_map.get(principal) {
+            Some(ref assets) => match assets.get(asset_identifier) {
+                Some(values) => Some(values),
+                None => None,
+            },
+            None => None
+        }
+    }
+
+    pub fn get_fungible_token_ids(&self, principal: &PrincipalData) -> Vec<AssetIdentifier> {
```

Comment:
This and the next function don't appear to be used anywhere:

```
$ find src -name "*.rs" -exec grep "get_fungible_token_ids" {} /dev/null \;
src/vm/contexts.rs:    pub fn get_fungible_token_ids(&self, principal: &PrincipalData) -> Vec<AssetIdentifier> {
$ find src -name "*.rs" -exec grep "get_nonfungible_token_ids" {} /dev/null \;
src/vm/contexts.rs:    pub fn get_nonfungible_token_ids(&self, principal: &PrincipalData) -> Vec<AssetIdentifier> {
```

And so should be removed.

Reply:
Done